plug.vim

```diff
@@ -986,6 +986,10 @@ function! s:update_impl(pull, force, args) abort
   let s:clone_opt = get(g:, 'plug_shallow', 1) ?
         \ '--depth 1' . (s:git_version_requirement(1, 7, 10) ? ' --no-single-branch' : '') : ''
 
+  if(has('win32unix'))
```

Comment:
We don't use parentheses when they're not necessary. Can you remove the outer parentheses?